src/main/java/consoleCommands/FileCommands.java

```diff
@@ -0,0 +1,76 @@
+package consoleCommands;
+
+import exception.InvalidFileDataException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.FileNotFoundException;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class FileCommands {
+    public static final String TRUE = "1";
+    public static final String FALSE = "0";
```

Comment:
These variable names could trip the reader/maintainer, given that there is already keywords of the same name